docs/algorithms/afl/implementation_ja.md

```diff
@@ -9,8 +9,6 @@
 
 fuzzufにおいてAFLは `include/fuzzuf/algorithms/afl` および `algorithms/afl` 以下で実装されています。AFLは派生アルゴリズムを多く持つ性質上、ほとんどの構成クラスがテンプレートによって実装されます。したがって、それらのクラスの宣言・定義は全て `include/fuzzuf/algorithms/afl` 以下のヘッダーファイル内にあります。
 
-特に、それらのテンプレートクラスの宣言は `include/fuzzuf/algorithms/afl` 直下のファイル、定義は `include/fuzzuf/algorithms/afl/templates` 直下のファイルで行われています。これは、明確にfuzzufのコーディング規約で定まっているルールというわけではありません。AFLは、元々通常のクラスとして実装されていたものをテンプレートクラスとして実装し直しており、git上の差分を少なくするためにはそのような構成にせざるをえなかったという歴史的な経緯によります。今後のリファクタリングよって、`templates` サブディレクトリを削除し、ファイルをまとめることを検討しています。
```

Comment:
I think this paragraph has some degrees of importance and hence should not be removed.
Actually this is not written in TODO.md, and would be confusing. 

Considering that this `/templates/` sub-directory can be seen in other algorithms than AFL, I agree to remove this if we move this paragraph somewhere else by for example noting about this in TODO.md.

Reply:
[4edbf1a](https://github.com/fuzzuf/fuzzuf/pull/20/commits/4edbf1a9cf736fb6c34795f3d4975a3f35a17a1f) moves this paragraph to TODO.md.

Reply:
[6377ba0](https://github.com/fuzzuf/fuzzuf/pull/20/commits/6377ba0398b72c412dcfdec395452a9932674a47) reverts this change.